openoa/plant.py

```diff
@@ -1068,23 +1066,36 @@ def parse_asset_geometry(
         Ref: http://geopandas.org/projections.html
 
         Args:
-            reference_system (:obj:`str`, optional): Used to define the coordinate reference system (CRS).
+            reference_system (:obj:`str`, optional): Used to define the coordinate reference system
+                (CRS). If None is used, then the `metadata.reference_system` value will be used.
                 Defaults to the European Petroleum Survey Group (EPSG) code 4326 to be used with
                 the World Geodetic System reference system, WGS 84.
-            utm_zone (:obj:`int`, optional): UTM zone. If set to None (default), then calculated from
-                the longitude.
-            reference_longitude (:obj:`float`, optional): Reference longitude for calculating the UTM zone. If
-                None (default), then taken as the average longitude of all assets.
+            utm_zone (:obj:`int`, optional): UTM zone.  If None is used, then the
+                `metadata.utm_zone` value will be used. Defaults to the being calculated from
+                :py:attr:`reference_longitude`.
+            reference_longitude (:obj:`float`, optional): Reference longitude for calculating the
+                UTM zone. If None is used, then the `metadata.reference_longitude` value will be
+                used. Defaults to the mean of `asset.longitude`.
 
         Returns: None
             Sets the asset "geometry" column.
         """
+        # Check for metadata inputs
+        if utm_zone is None:
+            utm_zone = self.metadata.utm_zone
+        if reference_longitude is None:
+            reference_longitude = self.metadata.reference_longitude
+        if reference_system is None:
+            reference_system = self.metadata.reference_system
```

Comment:
I think we have a similar thing going on here - except the reference default reference system will be changed for the whole PlantData (and, hypothetically, any other PlantData which shares the same Metadata)

Reply:
This is data that shouldn't actually changed given it's the reference data, but I could see not resetting it if that's a real hangup.